src/python/pants/engine/fs.py

```diff
@@ -59,32 +59,29 @@ class PathGlobs:
   NB: this object is interpreted from within Snapshot::lift_path_globs() -- that method will need to
   be aware of any changes to this object's definition.
   """
-  include: Tuple[str, ...]
-  exclude: Tuple[str, ...]
+  globs: Tuple[str, ...]
   glob_match_error_behavior: GlobMatchErrorBehavior
   conjunction: GlobExpansionConjunction
   description_of_origin: str
 
   def __init__(
     self,
-    include: Iterable[str],
-    exclude: Iterable[str] = (),
+    globs: Iterable[str],
     glob_match_error_behavior: GlobMatchErrorBehavior = GlobMatchErrorBehavior.ignore,
     conjunction: GlobExpansionConjunction = GlobExpansionConjunction.any_match,
     description_of_origin: Optional[str] = None,
   ) -> None:
     """
-    :param include: globs to match, e.g. "foo.txt" or "**/*.txt"
-    :param exclude: globs to exclude, in the same style as the args to `include`
+    :param globs: globs to match, e.g. `foo.txt` or `**/*.txt`. To exclude something, prefix it
+                  with `!`, e.g. `!ignore.py`.
```

Comment:
This is actually *easier* to explain than `include` and `exclude` separately. Love this change!